Presentations/02-intro-to-R-programming.Rmd

```diff
@@ -948,15 +915,19 @@ knitr::include_app("https://rrmaximiliano.shinyapps.io/learnr-app/")
 
 # Custom functions
 
-### Exercise 8: Putting it all together!
+.exercise[
+
+### Exercise 8 `r fa("keyboard")`
 
 1. Use tidyverse's `select()` to select the columns `health_life_expectancy` and `freedom` in `whr`
 
-2. Use `sapply()` combined with the `zscore` function to get the z-score of these two columns
+2. Use `map()` combined with the `zscore` function to get the z-score of these two columns
```

Comment:
In here, I think it would be more useful to create new variables with the outcome of these functions. But I see it may be too big  a step, so up to you @luisesanmartin .

Reply:
Nice suggestion, I changed exercise 8 accordingly and added a couple of slides explaining single vs double-brackets indexing in lists, which is necessary to extract the actual vectors from the result of `map()`.
This is in 185dcc8